Aztec/Classes/TextKit/TextView.swift

```diff
@@ -135,14 +135,16 @@ open class TextView: UITextView {
     ///
     fileprivate var textAttachmentImageProvider = [TextViewAttachmentImageProvider]()
 
-    // MARK: - Properties: Formatting
```

Comment:
Why are we removing the mark?

Reply:
PS: if we want to rename this I'm fine, but it's a good one since all of it's content is related to formatting.

Reply:
Related to the change few lines above. This is part of the [delegates section](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/794/files/3836eb1eff0bf936c76da6b30e65a45b6b816463#diff-8d8015f3f021c0252c1569e4965dce80L127), so it's a bit of redundant